.delivery/build-cookbook/attributes/default.rb

```diff
@@ -2,3 +2,8 @@
 # systemd for habitat's services.
 default['ciainfra']['image_id'] = 'ami-7ac6491a'
 default['ciainfra']['security_groups'] = ['sg-9d9223f9']
+
+default['delivery']['project_apps'] = %w(
```

Comment:
You can read up more about the _Project Applications_ pattern at:
https://docs.chef.io/delivery_truck.html#project-applications